bitnami/tomcat/templates/jmx-configmap.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and .Values.metrics.jmx.enabled .Values.metrics.jmx.config (not .Values.metrics.jmx.existingConfigmap) }}
```

Comment:
I would add a validation in the NOTES.txt to check that if `.Values.metrics.jmx.enabled` is `true`, either `metrics.jmx.config` or `metrics.jmx.existingConfigmap` must be set.

Reply:
new funtion added to _helper to check that and called into NOTE.txt. You can reviewing if it's fine for you